app/src/main/java/edu/wisc/ece/pinpoint/pages/map/MapFragment.java

```diff
@@ -179,8 +275,22 @@ private void handleUndiscoveredPinClick(Marker marker) {
             if (LocationDriver.isCloseEnoughToFindPin(
                     new LatLng(userLoc.getLatitude(), userLoc.getLongitude()),
                     marker.getPosition())) {
+                PinSource pinSource;
+                switch (marker.getSnippet()){
+                    case "DEV":
+                        pinSource = PinSource.DEV;
+                        break;
+                    case "NFC":
+                        pinSource = PinSource.NFC;
+                        break;
+                    default:
+                        // Do not store friend enum in cloud,
```

Comment:
Do we want to be storing NFC status in the cloud either?

Reply:
I'll find out what works best for nfc in my next ticket